nebula.proto

```diff
@@ -62,6 +62,7 @@ message NebulaHandshakeDetails {
   uint32 ResponderIndex = 3;
   uint64 Cookie = 4;
   uint64 Time = 5;
+  uint32 CertVersion = 8;
```

Comment:
thoughts on making this an enum to match cert.go?

Reply:
It would be a cyclical import for the `cert` package or else we'd have to do silly type conversions between the two packages.

I will say using `cert.Version*` for protocol level stuff gave me the ick.